README.md

```diff
@@ -21,7 +21,13 @@ The first command above will install pytorch-cpu, which, as the name suggests,
  does not have cuda support. 
  
 `Note: inltk is currently supported only on Linux with Python >= 3.6`
- 
+
+### Intallation on windows (experimental)
+```bash
+pip3 install https://download.pytorch.org/whl/cpu/torch-1.1.0-cp36-cp36m-win_amd64.whl
+pip3 install torchvision
```

Comment:
@ibrahiminfinite Is there any specific reason we need to install torchvision? I think it should work without it as well.

Reply:
@goru001 its been removed.